luaui/Widgets/api_blueprint.lua

```diff
@@ -1,5 +1,16 @@
 local widget = widget ---@type Widget
 
+local BpDefs = VFS.Include("luaui/Include/blueprint_substitution/definitions.lua")
+VFS.Include("luaui/Include/blueprint_substitution/logic.lua")
+
+local isDevModeParam = Spring.GetGameRulesParam('isDevMode')
+local ENABLE_REPORTS = (isDevModeParam == 1 or isDevModeParam == true or isDevModeParam == "1")
+local DEBUG_MODE = ENABLE_REPORTS
+
+local reportFunctions = nil
+
+local activeBlueprint = nil
```

Comment:
moved this to the top of the file to reflect that it is now "global state", insomuch as it is accessed by cmd_blueprint via a getter.